assets/components/molecules/question/question.scss

```diff
@@ -77,7 +78,7 @@ $animation: 0.2s;
 
   // checked animation and styles
   .custom-control-input:checked + .custom-control-label {
-    @extend .custom-control-label:hover;
+    @extend .custom-control-label, :hover;
```

Comment:
I'm not sure about this part.

| Before  | After  | 
|---|---|
|  ![Screenshot from 2022-11-03 09-14-53](https://user-images.githubusercontent.com/2843501/199675886-ec7073cd-2bd5-4a75-9d67-550194b69a81.png) |  ![Screenshot from 2022-11-03 09-16-11](https://user-images.githubusercontent.com/2843501/199675919-b7340210-e611-4663-b8f5-3ee269fd7d42.png) | 

http://localhost:3000/#/molecules/question

Reply:
Nice catch! Need to put on my thinking hat.

Reply:
Yep, that was wrong. Fixed in the last pushed version, thanks for spotting.